comit/src/network/orderbook/order.rs

```diff
@@ -50,6 +52,26 @@ impl FromStr for OrderId {
     }
 }
 
+/// The position of the maker for this order. A BTC/DAI buy order,
+/// also described as an order that buys the trading pair BTC/DAI,
+/// means that the maker buys the base currency (in this case BTC) in
+/// return for DAI. A sell order means that the maker sells BTC and
+/// receives DAI.
+///
+/// Please note: we do not set the base currency to 1 and use rate
+/// (i.e., quote currency) and amount as is commonly done in Forex
+/// trading. We use the amounts of each currency to determine the
+/// rate.
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum Position {
+    Buy,
+    Sell,
+}
+/// The amount of DAI in wei for one satoshi of BTC
+// todo: deserialise from decimal strings
+pub type Rate = u64;
```

Comment:
When you come across `Rate` in the code, it gives you a false impression of safety that is associated with newtypes whereas in reality, any u64 will do and the compiler won't complain.

I find this use of type aliases does more harm than good. Type aliases are useful if you want to shorten a long type but this kind of "translation" between primitive types and domain types should either happen as part of a newtype or not at all.